src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
`mudlet` table is already used for storing that a mapper script exists, so I'm guessing a lot of profiles would have it already, that's why I think `mudlet.supportscoroutines` is reasonable.

Look at this from the most important POV in this context - the scripters - writing `if mudlet.supports and mudlet.supports.coroutines then` is just bulky.